framework/celery_tasks/routers.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+from website.settings import DEFAULT_QUEUE, LOW_QUEUE, MED_QUEUE, HIGH_QUEUE
+
+LOW_PRI_MODULES = {
```

Comment:
I might move these into the settings file if it won't cause a horrendous circular import. All these have to be modified when a new task is added, just like the list of celery task. It might be more discoverable if they're all in the same place. Just a suggestion.